python/cudf/dataframe/columnops.py

```diff
@@ -107,7 +120,7 @@ def column_empty_like(column, dtype, masked, newsize=None):
 def column_empty(row_count, dtype, masked, categories=None):
     """Allocate a new column like the given row_count and dtype.
     """
-    dtype = np.dtype(dtype)
+    dtype = pd.api.types.pandas_dtype(dtype)
```

Comment:
Did not know this function exists, awesome! We should create a cuDF utility wrapping this function because there will be differences for things like datetimes, but we can do that in a later PR.